tests/testthat/test-integration-fims-estimation-with-wrappers.R

```diff
@@ -257,183 +290,166 @@ test_that("estimation test of fims using wrapper functions", {
-        AgeComp = "DmultinomDistribution",
-        LengthComp = "DmultinomDistribution"
-      )
-    ),
-    survey1 = list(
-      selectivity = list(form = "LogisticSelectivity"),
-      data_distribution = c(
-        Index = "DlnormDistribution",
-        AgeComp = "DmultinomDistribution",
-        LengthComp = "DmultinomDistribution"
-      )
+  # Define fleet and survey specifications
+  fleet1 <- survey1 <- list(
+    selectivity = list(form = "LogisticSelectivity"),
+    data_distribution = c(
+      Index = "DlnormDistribution",
+      AgeComp = "DmultinomDistribution"
     )
   )
 
-  lengthcomp_parameters <- fims_data |>
+  # Run FIMS model with following steps
+  # * Create default parameters with fleet1 and survey1 specifications
+  # * Update parameters if any modifications are provided
+  # * Initialize FIMS with the provided data and parameters
+  # * Fit the FIMS model with optimization enabled
+  agecomp_fit <- data_age_comp |>
     create_default_parameters(
-      fleets = fleets,
-      recruitment = list(
-        form = "BevertonHoltRecruitment",
-        process_distribution = c(log_devs = "DnormDistribution")
-      ),
-      growth = list(form = "EWAAgrowth"),
-      maturity = list(form = "LogisticMaturity")
-    )
+      fleets = list(fleet1 = fleet1, survey1 = survey1)
+    ) |>
+    update_parameters(modified_parameters = modified_parameters) |>
+    initialize_fims(data = data_age_comp) |>
+    fit_fims(optimize = TRUE)
 
-  modified_parameters <- list(
-    fleet1 = list(
-      Fleet.log_Fmort.value = log(om_output_list[[1]][["f"]])
-    ),
-    survey1 = list(
-      LogisticSelectivity.inflection_point.value = 1.5,
-      LogisticSelectivity.slope.value = 2,
-      Fleet.log_q.value = log(om_output_list[[1]][["survey_q"]][["survey1"]])
-    ),
-    recruitment = list(
-      BevertonHoltRecruitment.log_rzero.value = log(om_input_list[[1]][["R0"]]),
-      BevertonHoltRecruitment.log_devs.value = om_input_list[[1]][["logR.resid"]][-1],
-      BevertonHoltRecruitment.log_devs.estimated = FALSE,
-      DnormDistribution.log_sd.value = om_input_list[[1]][["logR_sd"]]
-    ),
-    maturity = list(
-      LogisticMaturity.inflection_point.value = om_input_list[[1]][["A50.mat"]],
-      LogisticMaturity.inflection_point.estimated = FALSE,
-      LogisticMaturity.slope.value = om_input_list[[1]][["slope.mat"]],
-      LogisticMaturity.slope.estimated = FALSE
-    ),
-    population = list(
-      Population.log_init_naa.value = log(om_output_list[[1]][["N.age"]][1, ])
-    )
-  )
-
-  parameters <- lengthcomp_parameters |>
-    update_parameters(
-      modified_parameters = modified_parameters
-    )
+  clear()
 
-  parameter_list <- initialize_fims(
-    parameters = parameters,
-    data = fims_data
+  # Compare FIMS results with model comparison project OM values
+  validate_fims(
+    report = agecomp_fit@report,
+    sdr = agecomp_fit@estimates,
+    sdr_report = agecomp_fit@estimates,
```

Comment:
How come the sdr_report is the estimates table?

Reply:
I’ve updated the tests to use get_report() and get_estimates() when working with the wrapper functions.